meta-ostro/classes/image-dsk.bbclass

```diff
@@ -38,6 +39,74 @@ inherit pythonnative
 # suffix. Probably should be moved to ostro-os.conf eventually.
 IMAGE_NAME_SUFFIX = ""
 
+create_ova() {
+
+  if [ "x${MACHINE}x" != "xintel-corei7-64x" ]; then
+    exit 0
+  fi
+
+  # 64 bit linux with sata bus
+  OS_TYPE="Linux_64"
+  STORAGE_NAME="SATA"
+  STORAGE_BUS_TYPE="sata"
+
+  VM_NAME="${IMAGE_NAME}"
+  # 512 should be large enough value to run Ostro (Galileo has 256MB RAM for example)
+  # but small enough value that even a modest host machine can afford it
+  RAM="512"
+  # min allowed - as we do not run graphical environment, it should be enough
+  VRAM="12"
+  # As above with RAM, 2 cores should be enough for Ostro without crippling the host machine
+  CPU_CORE_COUNT=2
+  # required for successful boot
+  FIRMWARE="efi"
+  IOAPIC="on"
+
+  # default settings for hard drive
+  STORAGE_PORT="0"
+  STORAGE_DEVICE="0"
+  STORAGE_DEVICE_TYPE="hdd"
+
+  
+  RAW_IMAGE="${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.dsk"
+  VIRTUALBOX_IMAGE="${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.vmdk"
```

Comment:
I'm not following the thinking to be honest. That VirtualBox manual page is really about how to give raw access to the host machine HDD. They seem to use an artefact from VMDK to achieve that.

You always need a disk image to be provided as part of an .ova appliance, creating a vmdk using an obscure feature of VirtualBox seems awkward given we have the ability to properly converting our .dsk into .vdi (and also given that the new steps don't really make us gain anything, we're still converting our .dsk image to something else).

What would make sense in my opinion is that we no longer *publish* .vdi images once .ova are available.